### PR TITLE
Create nodejs back-end server and api

### PR DESCRIPTION
**Pull request by geekysuman**
@inovizz please review this..

### CODE REVIEW

File: server/routes.js [hunk 1]
```diff
@@ -1,32 +1,337 @@
...
+
+if (typeof web3 !== 'undefined') {
+  var web3 = new Web3(web3.currentProvider)
+} else {
+  var web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:8545'))
+}
+
+LMS.setProvider(web3.currentProvider)
+
+// use following command to generate LMS.json
```
Comment (inovizz) [inline server/routes.js]: Can you remove this commented piece of code, since we don't need it for any future reference.

File: server/routes.js [hunk 2]
```diff
@@ -1,32 +1,337 @@
...
+				res.json({ result : result.args });
+			}
+		});
+	});*/
+
+
+	// POST API calls
+
+	// This API is not tested after the changes
+	// api.route('/create_account')
```
Comment (inovizz) [inline server/routes.js]: if we comment out this create_account POST api call then account creation feature would stop working after merge. So please uncomment it for now. We need to improve it by using web3.personal.newAccount API.

File: server/routes.js [hunk 3]
```diff
@@ -1,32 +1,337 @@
...
+	api.get('/books', (req, res) => {
+		console.log("Displaying all books data");
+		lms.getAllBooks()
+		.then((message) => {
+			res.json({ message });   
+		})
+		.catch((err) => {
+			console.log(err);
+		});
+		// res.json({ message: lms.getAllBooks()});
```
Comment (inovizz) [inline server/routes.js]: Please remove this commented line of code and there commented lines in other API calls as well, so remove them as well.

Comment (geekysuman) [inline server/routes.js]: updated ... please check once. @inovizz

**Pull request merged**